docs/specifications/tests/Basic-TP/basic01.md

```diff
@@ -1,199 +1,376 @@
+
+
+## Intercase dependencies
 
-[INCONSISTENT_DELEGATION]: #outcomes
+None.
 
-[Get-Parent-Zone]: ../Methods.md#method-get-parent-zone
 
+## Terminology
+
+* "Direct Subdomain" - Domain A is considered to be a "direct Subdomain" to
+  domain B if domain A is just the addition of one label at the least significant
+  (left) side, e.g. "foo.domain.com" is a direct subdomain to "domain.com".
+
+* "DNS Lookup" - The term is used when a recursive lookup is used, though
+any changes to the DNS tree introduced by an [undelegated test] must be
+respected. Compare with "[Send]".
+
+* "Non-Referral" - See "[Referral]".
+
+* "Referral" - A DNS response with [RCODE Name] NoError, AA flag unset and NS
+records in the authority section. The answer section is empty or with CNAME
+record or records. If the query type is CNAME, then the answer section must be
+empty (does not apply to this test case). The additional section may contain
+address records (A and AAAA) for the name server names from the NS (glue
+records).
+
+* "Send" - The term "send" (to an IP address) is used when a DNS query is sent to
+a specific name server. Compare with "[DNS Lookup]".
+
+
+[Argument list]:                                                  https://github.com/zonemaster/zonemaster-engine/blob/master/docs/logentry_args.md
+[B01_CHILD_FOUND]:                                                #Summary
+[B01_CHILD_IS_ALIAS]:                                             #Summary
+[B01_CHILD_NOT_EXIST]:                                            #Summary
+[B01_INCONSISTENT_ALIAS]:                                         #Summary
+[B01_INCONSISTENT_DELEGATION]:                                    #Summary
+[B01_INCONSISTENT_DELEGATION]:                                    #Summary
+[B01_NO_CHILD]:                                                   #Summary
+[B01_PARENT_FOUND]:                                               #Summary
+[B01_PARENT_UNDETERMINED]:                                        #Summary
+[B01_UNEXPECTED_NS_RESPONSE]:                                     #Summary
+[Basic03]:                                                        basic03.md
+[CRITICAL]:                                                       ../SeverityLevelDefinitions.md#critical
+[Direct Subdomain]:                                               #terminology
+[DNS Lookup]:                                                     #terminology
+[DNS Query and Response Defaults]:                                ../DNSQueryAndResponseDefaults.md
+[DNS Query]:                                                      ../DNSQueryAndResponseDefaults.md#default-setting-in-dns-query
+[DNS Response]:                                                   ../DNSQueryAndResponseDefaults.md#default-handling-of-a-dns-response
+[ERROR]:                                                          ../SeverityLevelDefinitions.md#error
+[Get parent zone]:                                                ../MethodsV2.md#method-get-parent-zone
```

Comment:
broken link

Reply:
Probably because the other document was merged later. I'll rebase this.

Reply:
It works after rebasing. Or rather cherry-picking.